decentraland/explorer-website
# feat: new loading

<img width="1440" alt="Screen Shot 2022-01-10 at 16 25 55" src="https://user-images.githubusercontent.com/208789/148826852-de2093f4-29e7-45bf-9b66-5313a580978b.png">

<img width="692" alt="Screen Shot 2022-01-10 at 16 24 51" src="https://user-images.githubusercontent.com/208789/148826876-b7dc5c2e-4dce-47d7-8620-9a69fe2ba014.png">


# How to test it

- Click on `Play using wallet`
- Once you get the sign request go back to the explorer windows (do not confirm or cancel)
- Reload
- You should see the `Play using wallet` expanded and with your wallet selected
